app/ante/eth.go

```diff
@@ -152,25 +148,25 @@ func (avd EthAccountVerificationDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx
 
 		// check whether the sender address is EOA
 		fromAddr := common.BytesToAddress(from)
-		codeHash := avd.evmKeeper.GetCodeHash(fromAddr)
-		if codeHash != common.BytesToHash(evmtypes.EmptyCodeHash) {
+		acct, err := avd.evmKeeper.GetAccount(ctx, fromAddr)
```

Comment:
GetCodeHash is removed, here
codeHash != common.BytesToHash(evmtypes.EmptyCodeHash)
why this check is not necessary?

Reply:
It is checked in the lines below.